src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const keysValues = sourceString.split(';');
+
+  const cssObject = {};
+
+  keysValues.forEach((keyValue) => {
+    const trimmedKeyValue = keyValue.trim();
+
+    if (trimmedKeyValue) {
+      const parts = trimmedKeyValue.split(':');
+
+      const property = parts[0].trim();
+
+      const value = parts[1].trim();
+
+      cssObject[property] = value;
+    }
+  });
```

Comment:
The 'forEach' loop is appropriate here as it does not return a new array, but the code inside the loop could be simplified. You can destructure the 'keyValue' directly in the parameter list of the 'forEach' callback to avoid extra lines of code and improve readability.